cocos/tiledmap/tiled-layer.ts

```diff
@@ -88,7 +94,10 @@ export class TiledLayer extends UIRenderer {
     protected _userNodeMap: { [key: string]: TiledUserNodeData } = {};// [id] = node;
     protected _userNodeDirty = false;
 
-    // store the layer tiles node, index is caculated by 'x + width * y', format likes '[0]=tileNode0,[1]=tileNode1, ...'
+    /**
+     * @en store the layer tiles node, index is caculated by 'x + width * y', format likes '[0]=tileNode0,[1]=tileNode1, ...'.
```

Comment:
store -> Stores
- Should use capital character of the first word
- Must use the third person

Reply:
please check other api docs too